test/methods.test.js

```diff
@@ -184,15 +184,15 @@ const resourceMock = nock(config.api)
     }
   })
 
-test('create a dataset', async (t) => {
+test('action create a dataset', async (t) => {
   const client = new Client(
     config.authToken,
     config.organizationId,
     config.datasetId,
     config.api
   )
 
-  const response = await client.put('package_create', { "name": "my_dataset", "owner_org": "myorg"})
+  const response = await client.action('package_create', { "name": "my_dataset", "owner_org": "myorg"})
```

Comment:
In `action.test.js` there is already tests for that, I think we don't need this.

Reply:
@mariorodeghiero you haven't changed this in your last commit.